main.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# Create methods that do the following:
+# List all books.
+# List all people.
+# Create a person (teacher or student, not a plain Person).
+# Create a book.
+# Create a rental.
+# List all rentals for a given person id.
```

Comment:
- [ ] I suggest you minimize the use of commented codes as it makes your codebase messy and unprofessional. Kindly remove all these comments and just summarise what you want to have here in one line instead of having several lines of commented codes.